package.json

```diff
@@ -1,17 +1,15 @@
 {
   "name": "chase-car-dashboard",
-  "version": "4.2.0",
+  "version": "5.0.0",
```

Comment:
I agree with this version change, but it has to match Backend's version, so can you update Backend/setup.py's version to be 5.0.0?